src/main/index.ts

```
@@ -6,8 +6,10 @@ import { ILogObj, Logger } from 'tslog'
 import setupIpcHandlers from './handlers/IpcHandler'
 import setupProtocolHandlers from './handlers/ProtocolHandler'
 import setupShortcutHandlers from './handlers/ShortcutHandler'
+import { parseCliArgs } from './CliArgs'
 
 export let mainWindow!: BrowserWindow
+export let cliArgs = parseCliArgs(process.argv)
```

ESLint

[error] 'cliArgs' is never reassigned. Use 'const' instead.

(prefer-const)